lib/dl_auth_api_lib/dl_auth_api_lib/app.py

```diff
@@ -80,6 +81,9 @@ def create_app(self, app_version: str | None = None) -> web.Application:
         app.router.add_route("get", "/oauth/uri/google", google_views.GoogleURIView)
         app.router.add_route("post", "/oauth/token/google", google_views.GoogleTokenView)
 
+        app.router.add_route("post", "/oauth/uri/snowflake", snowflake_views.SnowflakeURIView)
```

Comment:
get mb?

Reply:
we have client_secret  as one of parameters